src/utils/png_reader.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PNG_READER_H
+#define PNG_READER_H
+
+#include "video/vga_palette.h"
+#include <stdbool.h>
+
+bool png_read_paletted(const char *filename, unsigned char *dst);
```

Comment:
Please do not create a function that starts with `png_`, as that's what the real libpng functions start with, such as png_read_png.

Reply:
We want to get rid of the sd_ prefix, since that is a remnant from old libsd days. Maybe read_png_ and write_png_ would be acceptable ?